include/nn/socket.h

```diff
@@ -15,7 +15,9 @@ namespace socket {
         u32 addr;
     };
 
+class Config;
     Result Initialize(void* pool, ulong poolSize, ulong allocPoolSize, int concurLimit);
+    Result Initialize(nn::socket::Config const&);
```

Comment:
```suggestion
    Result Initialize(void* pool, ulong poolSize, ulong allocPoolSize, int concurLimit);
```
My compiler doesn't know which overload should be hooked. There's probably a way to specify it, but it seems like this overload isn't used anywhere so it was easiest to fix it by just removing it.